src/containers/BookList.js

```diff
@@ -1,24 +1,27 @@
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
-import { createBookAction, removeBookAction } from '../actions';
 
 const BookList = ({ books }) => {
   const bookRows = books.map(book => (<Book key={`book-number-${book.bookId}`} book={book} />));
-
   return (
     <div>
```

Comment:
As stated above you in the specific requirements above, you need to implements `handleBookRemove` in this component. Hence you will be passing this method to the Book as a property hence the above code would look like this
`visible.map(book =>
 (<Book key={book-number-${book.bookId}} book={book} clickDelete={handleRemoveBook}/>));`

Reply:
Implemented this